sabakan/generator.go

```diff
@@ -503,21 +508,12 @@ func (g *Generator) increaseControlPlane() (*updateOp, error) {
 		return nil, nil
 	}
 
-	cps := make([]*Machine, len(g.controlPlanes))
-	for i, n := range g.controlPlanes {
-		cps[i] = g.machineMap[n.Address]
-	}
-
-	workers := make([]*Machine, len(g.workers))
-	for i, n := range g.workers {
-		workers[i] = g.machineMap[n.Address]
-	}
-
 	op := &updateOp{
-		name:    "increase control plane",
-		cps:     cps,
-		workers: workers,
+		name: "increase control plane",
 	}
+	g.nextControlPlanes = g.nodesToMachines(true)
```

Comment:
func name  nodesToMachines is slightly ambiguous. Adding the word "current" is better.